README.md

```diff
@@ -33,59 +33,73 @@ The directive will automatically re-run the $digest cycle on the scope, whenever
 
 It will also dispose of the autorun callback when the scope is destroyed.
 
-## Usage
+## Installation
 
-Install:
+Install, import, and include:
 ```
 $ npm install --save mobx-angularjs
 ```
 
-<br>
-
-Import `mobx-angularjs` and include the module:
 ```js
-import mobxAngular from 'mobx-angularjs';
+import mobxAngular from 'mobx-angularjs'
+
+angular.module('app', [ mobxAngular ])
+```
+
+### or
+
+Use CDN and include:
+
+```html
+<!-- development -->
+<script src="https://unpkg.com/mobx-angularjs@1.5.0/dist/mobx-angularjs.js"></script>
 
-angular.module('app', [ mobxAngular ]);
+<!-- production -->
+<script src="https://unpkg.com/mobx-angularjs@1.5.0/dist/mobx-angularjs.min.js"></script>
```

Comment:
We can use [unpkg](https://unpkg.com/#/) as a CDN since we are publishing to npm.